community/modules/compute/schedmd-slurm-gcp-v6-nodeset/main.tf

```diff
@@ -175,3 +175,21 @@ data "google_compute_reservation" "reservation" {
     # Add a validation that if reservation.project != var.project_id it should be a shared reservation
   }
 }
+
+data "google_compute_machine_types" "machine_types_by_zone" {
+  for_each = local.zones
+  filter   = format("name = \"%s\"", local.nodeset.machine_type)
+  zone     = each.value
+}
+
+locals {
+  machine_types_by_zone      = data.google_compute_machine_types.machine_types_by_zone
+  zones_without_machine_type = [for k, v in local.machine_types_by_zone : k if length(v.machine_types) == 0]
+}
+
+check "is_machine_type_available_in_zones" {
```

Comment:
Check is available starting Terraform 1.5.0 . AFAIK we are supporting terraform 1.2.0.

See `Makefile`: `MIN_TERRAFORM_VERSION=1.2 # for deploying modules`

Reply:
Yes, checks are available with only TF 1.5+. I had discussed a similar concern regarding TF version with Tom (that turned out to be a non-issue). He had clarified then that we should be able to use Terraform 1.5

@tpdownes would it be okay for us to bump up the minimum version to 1.5?

Reply:
LGTM, feel free to land if you get a green light in regard of terraform version.

Reply:
ack, will wait for Tom

Reply:
I agree that TF 1.5 is acceptable, but we should wait to put this in until https://github.com/GoogleCloudPlatform/cluster-toolkit/pull/2772 has been rolled back.

Reply:
Completed in #2961